src/components/Display.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Display({ value }) {
+  return (
+    <div className="display">
+      <span>{value}</span>
+    </div>
```

Comment:
- [ OPTIONAL ] I think it would be great if you add the calculation operation above in your calculator to let the user know what is the operation he do? so by this you are improving from the UI and to improve user experience 🔥💯🚀✅✅


![image](https://user-images.githubusercontent.com/58553711/228247829-79afce73-aa9a-4493-8f03-d7edfa038b5a.png)